docs/pages/v3/config.mdx

```diff
@@ -0,0 +1,56 @@
+Here is a simple example of an `open-next.config.ts` file: 
+This file need to be at the same place as your `next.config.js` file
+
+`server` in here could refer to a lambda function, a docker container, a node server or whatever that can support running nodejs code. (Even cloudflare workers in the future)
+
+For more information about the options here, just look at the source file
+
+```ts
+import type { BuildOptions } from 'open-next/types/open-next'
+const config = {
+  default: { // This is the default server, similar to the server-function in open-next v2
+    // You don't have to provide the below, by default it will generate an output 
+    // for normal lambda as in open-next v2
+    override: { 
+      wrapper: "aws-lambda-streaming", // This is necessary to enable lambda streaming
+      // You can override any part that is a `LazyLoadedOverride` this way
+      queue: () => Promise.resolve({
+        send: async (message) => {
+          //Your custom code here
+        }
+      })
+    },
+  },
+  functions: { // here we define the functions that we want to deploy in a different server
+    ssr: {
+      routes: ["app/api/isr/route", "app/api/sse/route", "app/api/revalidateTag/route"], // For app dir, you need to include route|page, no need to include layout or loading
```

Comment:
"include route|page", is the `page` for the `pages-dir`? That comment might be confusing.

Reply:
ah ok, it might be a little clearer if there was an example of the page, eg:
```typescript
routes: [
   "app/api/isr/route", "app/api/sse/route", "app/api/revalidateTag/route", // API routes
   "app/route1/page", "app/route2/page", "app/route3/page"                    // Pages
]
```

Reply:
I'll add this and merge